discussions/api.py

```diff
@@ -314,12 +316,20 @@ def get_membership_ids_needing_sync():
         QuerySet: query for the list of database ids for memberships that need to be synced
     """
     # Order by is_member (True before False) and updated_on (most recent first)
-    return PercolateQueryMembership.objects.filter(
+    memberships_quest = PercolateQueryMembership.objects.filter(
         query__source_type=PercolateQuery.DISCUSSION_CHANNEL_TYPE,
         user__profile__isnull=False,
         needs_update=True
     ).order_by('-is_member', '-updated_on').values_list("id", flat=True)
```

Comment:
If you want the query, I think you need to remove the `.values_list("id", flat=True)`